nvdaHelper/vbufBase/storage.cpp

```diff
@@ -579,6 +579,13 @@ VBufStorage_textFieldNode_t*  VBufStorage_buffer_t::addTextFieldNode(VBufStorage
 		delete textFieldNode;
 		return NULL;
 	}
+	if (needsStrip && subStart>0) {
+		// There are characters stripped from the start of the text.
+		// We save these characters in an attribute, required for calculating offsets in IA2Text.

```

Comment:
nit: This comment makes it seem like we're saving the characters themselves, not the number of characters. Please clarify.